src/main/java/net/neoforged/neoforge/common/NeoForgeMod.java

```diff
@@ -209,6 +209,23 @@ public class NeoForgeMod {
      */
     public static final Holder<Attribute> BLOCK_REACH = ATTRIBUTES.register("block_reach", () -> new RangedAttribute("neoforge.block_reach", 4.5D, 0.0D, 1024.0D).setSyncable(true));
 
+    /**
+     * Grants the player the ability to use creative flight when not in creative mode.
+     * Anything above zero allows flight.
+     * <p></p>
```

Comment:
Should be `<p>` only. Either between the lines like
```
A
<p>
B
```
or at the beginning of the new line:
```
A

<p>B
```